libs/components/src/dialog/simple-dialog/simple-dialog.stories.ts

```diff
@@ -82,3 +82,19 @@ export const ScrollingContent: Story = {
     useDefaultIcon: true,
   },
 };
+
+export const TextOverlow: Story = {
```

Comment:
Guess I missed this in the original PR

```suggestion
export const TextOverflow: Story = {
```

Reply:
Never heard of overlow before???